sdv/constraints/utils.py

```diff
@@ -194,3 +194,12 @@ def get_datetime_diff(high, low, dtype='O'):
     diff_column = diff_column.astype(np.float64)
     diff_column[nan_mask] = np.nan
     return diff_column
+
+
+def create_unique_name(name, list_names):
```

Comment:
One thing to note is that our utils module is technically public, so we should be careful about removing things/moving them around (I know I did that in the past 😅 ). In this case, it should be fine since we haven't done a release with this method yet. However, if we think it will ever be useful to have at a higher level (use outside of constraints), maybe we should leave it in sdv.utils

Reply:
The reason I did the change is because test where in one place (tests/unit/constraints/test_utils.py) and the code in sdv/utils.py so it had to go either one place or another 👍🏻 now should be fixed.